docs/src/main/paradox/scala/http/routing-dsl/routes.md

```diff
@@ -110,4 +112,15 @@ Here five directives form a routing tree.
 
 Route 3 can therefore be seen as a "catch-all" route that only kicks in, if routes chained into preceding positions
 reject. This mechanism can make complex filtering logic quite easy to implement: simply put the most
-specific cases up front and the most general cases in the back.
\ No newline at end of file
+specific cases up front and the most general cases in the back.
+
+## Sealing a Route
+
+As described in @ref[Rejections](rejections.md#rejections-scala) and @ref[Exception Handling](exception-handling.md#exception-handling-scala),
+there are generally two ways to handle rejections and exceptions.
+
+ * Bring rejection/exception handlers into implicit scope at the top-level
+ * Supply handlers as arguments to @ref[handleRejections](directives/execution-directives/handleRejections.md#handlerejections) and @ref[handleExceptions](directives/execution-directives/handleExceptions.md#handleexceptions) directives 
+
+In the first case your handlers will be "sealed", (which means that it will receive the default handler as a fallback for all cases your handler doesn't handle itself) 
+and used for all rejections/exceptions that are not handled within the route structure itself.
```

Comment:
Also explain that you can `Route.seal()` a route if you want to apply some modification on the outgoing HttpResponse that resulted from a rejection. Say, you always want to add some headers. But a rejection gets a default HttpResponse. So you want to `addHeaders { Route.seal( something where the rejection happens ) }`, since the addHeaders thingy would then work on the "error response" as well :)